src/omv/modules/py_image.c

```diff
@@ -768,14 +765,24 @@ static mp_obj_t py_image_unpack(uint n_args, const mp_obj_t *pos_args, mp_map_t
         }
     }
 
+    size_t shape[ULAB_MAX_DIMS];
+    size_t strides[ULAB_MAX_DIMS];
     int channels;
+    int ndim;
+
     switch (image->pixfmt) {
         case PIXFORMAT_GRAYSCALE: {
+            memcpy(shape, (size_t []) {0, 0, image->h, image->w}, sizeof(shape));
```

Comment:
```suggestion
            shape = (size_t []) {0, 0, image->h, image->w};
```
This should compile.

Reply:
It does not. I tried it this way first.

/home/kwagyeman/github/openmv/src/omv/modules/py_image.c:775:19: error: assignment to expression with array type
  775 |             shape = (size_t []) {0, 0, image->h, image->w};

Reply:
sad, it works for structs only.